build/linux/installer/scripts/livenessprobe.sh

```diff
@@ -85,9 +85,14 @@ fi
 (ps -ef | grep telegraf | grep -v "grep")
 if [ $? -ne 0 ]
 then
- # echo "Telegraf is not running" > /dev/termination-log
- echo "Telegraf is not running (controller: ${CONTROLLER_TYPE}, container type: ${CONTAINER_TYPE})" > /dev/write-to-traces  # this file is tailed and sent to traces
- # exit 1
+ if [ "${CONTROLLER_TYPE}" == "ReplicaSet" ] && [ ! -z "${SIDECAR_SCRAPING_ENABLED}" ] && [ "${SIDECAR_SCRAPING_ENABLED}" == "true" ]; then
```

Comment:
SIDECAR_SCRAPING_ENABLED enabled by default for AKS & Arc. I dont think this required to check. But telegraf in replicaset used if the customer enabled configmap with cluster level prometheus settings. you would need to activate & check telegraf if the customer enabled cluster level prometheus settings.   Did you check that this is working as expected?